### PR TITLE
use file constructor

### PR DESCRIPTION
**Pull request by cakofony**
Not sure how exactly to run this locally.
I didn't realize the file method (on project) wouldn't work correctly from a plugin :-/

### CODE REVIEW

File: src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy [hunk 1]
```diff
@@ -89,10 +89,10 @@ class ProcessorsPlugin implements Plugin<Project> {
         project.idea.module.scopes.PROVIDED.plus += [project.configurations.processor]
       }
 
-      project.idea.module.generatedSourceDirs += file(project.processors.sourceOutputDir)
-      project.idea.module.sourceDirs += file(project.processors.sourceOutputDir)
-      project.idea.module.generatedSourceDirs += file(project.processors.testSourceOutputDir)
-      project.idea.module.testSourceDirs += file(project.processors.testSourceOutputDir)
+      project.idea.module.generatedSourceDirs += new File(project.projectDir, project.processors.sourceOutputDir)
```
Comment (uschi2000) [inline src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy]: Try project.file() if you want.

Comment (cakofony) [inline src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy]: Updated, thanks!

**Comment by cjp39** [comment]
Any chance of a test this time around? This is clearly in need of one :/ Look at the existing integration tests for inspiration.

**Comment by cakofony** [comment]
Fixed. Test fails against broken master.

**Comment by cjp39** [comment]
I need to stop accepting contributions without tests :D Thanks!

**Comment by cjp39** [comment]
If you could write an integration test too, I'd really appreciate it. Checking the generated XML is sounding pretty reasonable at this point.

**Pull request merged**